init.rb

```diff
@@ -5,17 +5,24 @@
   name 'Redmine Issue Evm plugin'
   author 'Hajime Nakagama'
   description 'This is a plugin for Redmine. Earned value management using the ticket of redmine.'
-  version '3.4.1'
+  version '3.5'
   url 'https://github.com/momibun926/redmine_issue_evm'
   author_url 'https://github.com/momibun926'
 
   project_module :Issuevm do
     permission :view_evms, :evms => :index , :require => :member
-    permission :manage_evms, { :evmbaselines => [:edit, :destroy, :new, :create, :update, :index] }
+    permission :manage_evmbaselines, { :evmbaselines => [:edit, :destroy, :new, :create, :update, :index, :show, :history] }
+    permission :view_evm_baselines, { :evmbaselines => [:index, :history, :show]}
```

Comment:
Line is too long. [81/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside } missing.